test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -69,28 +58,31 @@ DROP TABLE IF EXISTS peer;
 DROP TABLE IF EXISTS role;
 )";
 
-        pqxx::connection connection(pgopt_);
+        pqxx::connection connection(config_->postgres().options());
         pqxx::work txn(connection);
         txn.exec(drop);
         txn.commit();
         connection.disconnect();
 
         cpp_redis::redis_client client;
-        client.connect(redishost_, redisport_);
+        client.connect(config_->redis().host, config_->redis().port);
         client.flushall();
         client.sync_commit();
         client.disconnect();
 
-        iroha::remove_all(block_store_path);
+        iroha::remove_all(config_->blockStorage().path);
       }
 
-      std::string pgopt_ =
-          "host=localhost port=5432 user=postgres password=mysecretpassword";
-
-      std::string redishost_ = "localhost";
-      size_t redisport_ = 6379;
+      std::unique_ptr<iroha::config::Config> config_;
```

Comment:
Can not find where `config_` is initialized. I think that the reason why `ametsuchi_test` fails. Please fix

Reply:
https://github.com/hyperledger/iroha/blob/96523cb7dddd8c2c76ea388cf443560c16e5f3b5/test/module/irohad/ametsuchi/ametsuchi_fixture.hpp#L42